source/core/types/ut_executable.tpb

```diff
@@ -96,6 +97,21 @@ create or replace type body ut_executable is
   begin
     if self.is_defined() then
       l_start_transaction_id := dbms_transaction.local_transaction_id(true);
+      
+      l_client_info := case
+                         when a_item is of(ut_test) then
+                          'Test: '
+                         else
+                          'Suite: '
+                       end || a_item.name || case
```

Comment:
but then most likely users will see preparation procedures cause they tend to length more then actually tests. This is why I've chosen such approach